envoy-control-services/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/services/ServicesState.kt

```diff
@@ -3,30 +3,32 @@ package pl.allegro.tech.servicemesh.envoycontrol.services
 typealias ServiceName = String
 
 data class ServicesState(
-    val serviceNameToInstances: Map<ServiceName, ServiceInstances> = emptyMap()
+    val serviceNameToInstances: MutableMap<ServiceName, ServiceInstances> = mutableMapOf()
```

Comment:
This class has a lack of encapsulation. This wasn't big deal before but now `serviceNameToInstances` is mutable. Please consider to encapsulate it with unmutableMap interface

Reply:
done